src/renderer/i18n/locales/it-IT.js

```diff
@@ -249,6 +254,14 @@ export default {
     TOOLTIP: 'La nuova versione ({version}) è stata rilasciata!'
   },
 
+  APP_SIDEMENU_PLUGIN_CONFIRMATION: {
+    QUESTION: 'Sei sicuro di voler abilitare il sistema di plugin?',
+    NO: 'No, tienilo disabilitato',
+    YES: 'Sì, abilitalo',
+    NOTICE: 'ATTENZIONE: questo sistema è attualmente in BETA. Tutti i plugin non ufficiali sono utilizzati a proprio rischio. Al momento non possiamo verificare la sicurezza di qualsiasi plug-in di terze parti e tutti i test sulla sicurezza e la scoperta delle vulnerabilità sono responsabilità dell\'autore del plug-in. Accettando questo avviso, l\'utente riconosce di utilizzare plugin di terze parti a proprio rischio.',

```

Comment:
```suggestion
    NOTICE: 'ATTENZIONE: questo sistema è attualmente in BETA. Tutti i plugin non ufficiali sono utilizzati a proprio rischio. Al momento non possiamo verificare la sicurezza di qualsiasi plugin di terze parti e tutti i test sulla sicurezza e la scoperta delle vulnerabilità sono responsabilità dell\'autore del plugin. Accettando questo avviso, l\'utente riconosce di utilizzare plugin di terze parti a proprio rischio.',
```

Reply:
@dated
The word "_plugin_" also appears in other fields. At this point I'd change them all. Not just here. If you confirm I will proceed with the changes.

"plugin" ---> "plug-in"

Reply:
Don't worry about the comment above. Fixed. Ty